vendor/github.com/google/cel-go/ext/strings.go

```diff
@@ -520,7 +522,7 @@ func (lib *stringLib) CompileOptions() []cel.EnvOption {
 	if lib.version >= 3 {
 		opts = append(opts,
 			cel.Function("reverse",
-				cel.MemberOverload("reverse", []*cel.Type{cel.StringType}, cel.StringType,
+				cel.MemberOverload("string_reverse", []*cel.Type{cel.StringType}, cel.StringType,
```

Comment:
is this user-exposed, or an internal-only name? wanting to make sure the rename is safe / compatible. (looks like we're only using up to ext.StringsVersion(2), so maybe we're insulated from it anyway?)

Reply:
No, this is the internal name CEL uses for the overload.  The grammatical name is `reverse` (`cel.Function`'s 1st arg).  Kubernetes is on version 2 of the strings library, so isn't using this function yet.